packages/astro/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License Copyright (c) 2023 Liran Tal
```

Comment:
This made me realize that we don't have individual licenses currently for our public packages, which has been fixed in this commit:

https://github.com/triggerdotdev/trigger.dev/commit/b5db9f5e5cf121e37b6350bf28fb64dc81ec949b

Do you think you could update this LICENSE file to match the others with our copyright? I think our policy will be that if it's in the monorepo then the LICENSE should be under Trigger.dev. If that doesn't work for you, then it's also okay for it to live as a standalone package (although not under the `@trigger.dev` npm org.

Reply:
Yes, fixed!